app/code/Magento/Quote/Model/GuestCart/GuestPaymentMethodManagement.php

```diff
@@ -7,7 +7,6 @@
 
 use Magento\Quote\Api\PaymentMethodManagementInterface;
 use Magento\Quote\Api\GuestPaymentMethodManagementInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `get` and `set` methods without the fully qualified name